packages/gatsby-source-contentful/README.md

```diff
@@ -152,6 +152,14 @@ For example, to filter locales on only germany `localeFilter: locale => locale.c
 
 List of locales and their codes can be found in Contentful app -> Settings -> Locales
 
+**richText.includeEntryFields** and **richText.excludeEntryFields** [array][optional] [default: `undefined`]
+
+If you have entries that reference each other via their rich-text fields, Gatsby may get stuck in an infinite loop during build and throw a `Maximum call stack size exceeded` error. To prevent this, you can filter which fields should be included in entries referenced by the field. Set a list of field names to include or exclude using this option, and Gatsby will filter the fields of referenced entries accordingly.
```

Comment:
I landed a fix that should address these circular references. I think this description can be updated to reflect that.

With that in mind, is this feature still necessary? Useful?

Reply:
Thanks @pvdz! We have a ticket on our end to upgrade Gatsby to see if it fixes these issues. If it does, we'll close this PR. This should all happen in the next few weeks. Thanks again! Excited to try the fix :)